docs/cristin.ttl

```diff
@@ -319,8 +319,9 @@
     """ ;
     rdfs:range         xsd:date ;
     rdfs:domain        :Result, :VITENSKAPELIGARBEID_LOKAL_Class ;
-    :mapstonvafield    "createdDate when the domain is :Result" ;
-    :currently_ignored "true" .
+    :mapstonvafield    """  createdDate when the domain is :Result
```

Comment:
Probably we should remove the mapping descriptions since now we have the cucumber scenarios.